src/app/clusters/electrical-energy-measurement-server/EnergyReportingTestEventTriggerDelegate.h

```diff
@@ -0,0 +1,83 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/TestEventTriggerDelegate.h>
+
+namespace chip {
+
+/*
+ * These Test EventTrigger values can be used to fake meter reading data
+ *
+ * They are sent along with the enableKey (manufacturer defined secret)
+ * in the General Diagnostic cluster TestEventTrigger command
+ */
+enum class EnergyReportingTrigger : uint64_t
+{
+    // We use the Electrical Energy Measurement cluster ID as our TE trigger space
+    // Stop Fake readings
+    kFakeReadingsStop = 0x0091000000000000,
```

Comment:
```suggestion
    kFakeReadingsStop = 0x0091'0000'0000'0000,
```

and same below. I believe it may be easier to parse where the first hex diff starts.

Reply:
Done